internal/v5/auth_test.go

```diff
@@ -1089,7 +1089,7 @@ func (s *authSuite) TestGroupsForUserWithInvalidBody(c *gc.C) {
 	s.idM.body = "bad"
 	s.idM.contentType = "application/json"
 	groups, err := h.GroupsForUser("someone")
-	c.Assert(err, gc.ErrorMatches, `cannot get groups for someone: cannot unmarshal response: invalid character 'b' looking for beginning of value`)
+	c.Assert(err, gc.ErrorMatches, `GET .*: invalid character 'b' looking for beginning of value`)
```

Comment:
.* will actually be https://api.jujucharms.com/identity/u/fabrice/groups or similar.